### DISCUSSION TITLE
Deleting Parquet Files

### DISCUSSION THREAD

**Discussion by robertclaus** [duckdb/duckdb#6703] [Q&A]
Is there a way to delete parquet files (including on S3) using DuckDB directly? I currently use it to manage data in my parquet files, and often use it to save results to new files. It would be very helpful to be able to clean up those files without having to switch to a different tool.

Example that doesn't work in Python:
```
import duckdb
tablename = "test"
filename = "test.parquet"
con = duckdb.connect(database=":memory:")
con.execute(f"INSTALL parquet; LOAD parquet;")
con.execute(f"CREATE OR REPLACE TABLE '{tablename}'(i INTEGER, j INTEGER);")
con.execute(f"COPY {tablename} TO '{filename}' (FORMAT 'PARQUET', CODEC 'UNCOMPRESSED');")
con.execute(f"DROP TABLE '{filename}';")
```

The DROP TABLE doesn't seem to recognize files. With quotes there is a syntax error; without quotes it reads the `name.extension` as `schema.table` and can't find the schema.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
duckdb.ParserException: Parser Error: syntax error at or near "'test.parquet'"
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
duckdb.CatalogException: Catalog Error: Schema with name test does not exist!
```

**Comment by Mytherin** [comment]
This is not currently supported from within the system - but we could definitely consider adding support for it.
